core-tests/src/test/java/org/mule/runtime/core/routing/filters/ExpressionFilterTestCase.java

```diff
@@ -137,15 +136,14 @@ public void testRegexFilterWithAngleBrackets() {
 
   @Test
   public void testExceptionTypeFilter() {
-    ExpressionFilter filter = new ExpressionFilter("exception is java.lang.Exception");
+    ExpressionFilter filter = new ExpressionFilter("exception.getCause() is java.lang.Exception");
```

Comment:
I've added a comment in MULE-10519 (move everything away from ME) explaining that these change should be restored once that is gone and the wrapping no longer needed.